packages/shared/src/lib/redux/store.ts

```diff
@@ -9,7 +10,10 @@ import { configureStore } from '@reduxjs/toolkit';
  */
 export const _store = configureStore({
 	reducer: {
-		example: exampleReducer
+		example: exampleReducer,
+		post: postReducer,
```

Comment:
Since this state contains an array of posts, should we perhaps use plural?

I asked chatgtp:
> When naming state slices in a Redux store that hold arrays of objects, it's generally better to use plural names. This aligns with the idea that the state slice represents a collection of items. Using plural names makes it immediately clear that the state contains multiple entities of a given type.
>
>Why Plural is Preferred:
>- Clarity: Plural names suggest collections, while singular names often imply a single item.
>- Consistency: If other parts of your application follow this pattern, it reduces cognitive load.
>- Convention: The broader JavaScript and Redux community commonly follows this practice.

Reply:
> I asked chatgtp:

Do we have a real source?
Text generators can not be used as a reliable form of information in any shape of form.

For example, I prompted the Chat GPT the same prompt, but switched "plural" for "singular":
<img width="799" alt="Screenshot 2024-11-21 at 19 13 42" src="https://github.com/user-attachments/assets/691a777a-e0fd-4c84-a153-dcc7e0021083">

<img width="806" alt="Screenshot 2024-11-21 at 19 13 19" src="https://github.com/user-attachments/assets/7f2ff1b5-fa49-4470-b1ac-c1b6fac5b405">

Reply:
https://redux.js.org/style-guide#name-state-slices-based-on-the-stored-data

Redux's style guide doesn't explicitly mention plural or singular, but does implicitly indicate to use the plural form in their examples, so we can switch over to that.

Reply:
This is a very interesting outcome, I wasn't aware! GPT seems to be an amplifier for confirmation bias. This makes sense given that the program wants to give satisfying responses, to please the user.

Reply:
@Caleb-T-Owens it's worth keeping in mind I did not ask a leading question, I simply asked which one it preferred.